src/unity/python/turicreate/test/test_flexible_type.py

```diff
@@ -559,6 +559,13 @@ def test_type_conversions(self):
         with self.assertRaises(RuntimeError):
             sa_int_words.astype(int)
 
+    def test_hashable_dict_keys(self):
+        # testing valid sarray of inf's (inf is a float)
```

Comment:
What is `inf`?

Reply:
Inf and -inf are specific floating point values such that -Inf < x < Inf for other valid floating point values of x.

Reply:
However, this comment is wrong.